rslib/src/scheduler/answering/current.rs

```diff
@@ -64,15 +66,39 @@ impl CardStateUpdater {
         let ease_factor = self.card.ease_factor();
         let remaining_steps = self.card.remaining_steps();
         let memory_state = self.card.memory_state;
+        let elapsed_time = |last_ivl: u32| {
+            match self.card.queue {
+                CardQueue::Learn => {
+                    // Decrease reps by 1 to get correct seed for fuzz.
+                    // If the fuzz calculation changes, this will break.
+                    let mut card_copy = self.card.clone();
+                    card_copy.reps -= 1;
+                    let last_ivl_with_fuzz =
+                        self.learning_ivl_with_fuzz(get_fuzz_seed(&card_copy), last_ivl);
```

Comment:
It would be nice to factor this out into get_fuzz_seed_for_id_and_reps, so we don't need to clone the card.